source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -218,9 +183,7 @@ def make_location(self, obj):
         )
 
     def get_language_parameter(self, request):
-        language = request.get('_asgard_language', "english_us")
-        language_tag = getattr(Languages, language, Languages.english_us).value
-        return language_tag
+        return request.get('language', "fr-FR")
```

Comment:
Shouldn't you use the coverage's value if language isn't in request?

Reply:
:+1: